### PR TITLE
wicked: Add sysctl testsuite

### PR DESCRIPTION
**Pull request by cfconrad**
The first two basic tests:
 - t01_sysctl_load_files - (check load order of sysctl files)
 - t02_sysctl_check_defaults - (check sysctl defaults)

- Related ticket: https://github.com/openSUSE/wicked/pull/896 https://github.com/openSUSE/wicked/pull/895
- Verification run: 
  - ~http://openqa-3.wicked.suse.de/t74546~
  - ~https://openqa.suse.de/tests/7939462 (SLE-15-SP4)~

### CODE REVIEW

**Comment by cfconrad** [comment]
This is who it looks if it fails https://openqa.suse.de/tests/7939406#step/t01_sysctl_load_files/9 triggered with
```bash
openqa-clone-job --parental-inheritance --skip-download  --within-instance https://openqa.suse.de --skip-chained-deps 7924453 CASEDIR='https://github.com/cfconrad/os-autoinst-distri
-opensuse.git#pr_add_sysctl_tests' WICKED=sysctl WICKED_SKIP_VERSION_CHECK=1 | openqa-wait-job
```

**Comment by cfconrad** [comment]
Done from my side, VR's with latest code:
SLE-15-SP4
 * http://openqa.suse.de/t7943396 
 * http://openqa.suse.de/t7943398 
Tumbleweed:
 * http://openqa.wicked.suse.de/tests/46619
For wicked < 0.6.68
 * http://openqa.wicked.suse.de/tests/46623

**Comment by cfconrad** [comment]
I removed the `sysctl --system` comparison. 
VR: http://openqa.wicked.suse.de/tests/47881

**Comment by cfconrad** [comment]
any news?

File: tests/wicked/sysctl/sut/t01_sysctl_load_files.pm [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+}
+
+sub check {
+    my ($self, $ctx) = @_;
+
+    check_load_order();
+
+    # Check order of files in sysctl.d directories
+    for my $dir (@sysctl_d) {
+        assert_script_run("mkdir -p $dir && touch $dir/20-test.conf");
```
Comment (asmorodskyi) [inline tests/wicked/sysctl/sut/t01_sysctl_load_files.pm:74]: I think you can move `mkdir -p` just before this two loops and not waste time on typing this command over and over again

**Pull request merged**